src/gr11review/part2/Utility.java

```diff
@@ -1,5 +1,327 @@
+import java.util.*;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+
+
+/** 
+* Method 2: Add numbers together
+* @author: H. Chung
+*
+*/
 public class Utility {
-    
-}
\ No newline at end of file
+  public static int sumNumbers(String str) {
+    // Setting Variables
+    int intLength;
+    intLength = str.length();
+    int intCount;
+    int intSum = 0;
+    String strWord = "";
+    // Looping until every character is check through
+    for (intCount = 0; intCount < intLength; intCount++) {
+      // See if the value is a digit
+      if (Character.isDigit(str.charAt(intCount))) {
+        // Conditions to see if you can add a number in the string
+        if (intCount < intLength-1 && Character.isDigit(str.charAt(intCount+1))) {
+           // Finding a number within the sentence
+          strWord += str.charAt(intCount);
+        } else {
+          // Adding the numbers together
+          strWord += str.charAt(intCount);
+          intSum += Integer.parseInt(strWord);
+          strWord = "";
+        }
+      }
+    }
+    // return sum
+    return intSum;
+  }
+  /** 
+  * File IO 2: Reading words in file and picking the first alphabetically first word
+  * @author: H. Chung
+  *
+  */
+  public static String alphaWord (String filenametxt) throws IOException {
+    BufferedReader FileText = new BufferedReader(new FileReader(filenametxt));
+    // Setting Variables and Reading File
+    String strWord;
+    String strFinalWord;
+    int intCompare;
+    strWord = FileText.readLine();
+    strFinalWord = FileText.readLine();
+    // As long as it is not in the end of the file, continue reading it. 
+    while (strWord != null) {
+      // Compares the words in the text file
+      intCompare = strFinalWord.compareToIgnoreCase(strWord);
+      if (intCompare > 0) {
+        strFinalWord = strWord;
+      }
+      strWord = FileText.readLine();
+      }
+    // Closes file and returns the word that is alphabetically first
+    FileText.close();
+    return strFinalWord;
+  }
+  /** 
+  * Array 2: Print out numbers in array
+  * @author: H. Chung
+  *
+  */
+  public static int[] notAlone(int[] nums, int value) throws IOException {
+    // Setting Variables
+    int intCount = 0;
+    // As long as intCount is less than length of the array, it continues.
+    while (intCount < nums.length) {
+      intCount++;
+      // Condition to make it change the selected number to the highest amount on its left or right
+      if (intCount > 0 && intCount < nums.length - 1 && nums[intCount] == value) { 
+        if (nums[intCount] != nums[intCount - 1] && nums[intCount] != nums[intCount + 1]) {
+          nums[intCount] = Math.max(nums[intCount-1], nums[intCount+1]);
+        }
+      }
+    }
+    // return new array
+    return nums;
+  }
+  /** 
+  * Array 4: Whether or not the numbers are balanced, print true or false
+  * @author: H. Chung
+  *
+  */
+  public static boolean canBalance(int[] nums) throws IOException {
```

Comment:
@Haydenchung03 this method doesn't need to throw an exception (no native io happening in this method)